app/views/admin/splits/_variants.html.erb

```diff
@@ -1,7 +1,7 @@
 <article class="InfoCard sc-m-v--l">
   <div class="InfoCard-titleContainer">
     <h4>Variant Details</h4>
-    <%= link_to "Edit", new_admin_split_split_config_path(split), class: 'change-weights-link' %>
+    <%= link_to "Change Weights", new_admin_split_split_config_path(split), class: 'change-weights-link' %>
```

Comment:
🙌 👍 for more descriptive links. ive always been confused by the 5 different "edit" links on the split page - when I want to assign a user I never know which one to click 😂